scripts/travis/imap-setup.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh -e
+
+#
+# ### This is a script to setup a dovecot imap server for testing rml_imap
+#
+
+#
+# Declare the important path variables
+#
+
+# Directories used in the system
+BASEDIR=$(git rev-parse --show-toplevel)
+BUILDDIR="${BASEDIR}/build/ci/imap"
+TRAVISDIR="${BASEDIR}/scripts/travis"
+RUNDIR="${BUILDDIR}/dovecot_run"
+TLSRUNDIR="${BUILDDIR}/dovecot_tls_run"
```

Comment:
`dovecot/run/tls` if you really want it to be separate

Reply:
I am pretty sure that there needs to be separate run directories for the different running instances of dovecot

Reply:
they do....